app/models/user.rb

```diff
@@ -2,4 +2,10 @@ class User < ApplicationRecord
   has_secure_password
 
   validates :email, :user_name, presence: true, uniqueness: true
+  validates_format_of :email, with: /\A([^@\s]+)@((?:[-a-z0-9]+\.)+[a-z]{2,})\Z/i
+  validates :password, length: { in: 6..20 }, allow_nil: true
+
+  def active?
+    status == 'active'
```

Comment:
It's time. I've tried not to say that but you can use an enum for statuses example could be found here - https://github.com/skrix/lunch_app/blob/master/app/models/item.rb#L13 ;)